taxonomies/labels.txt

```diff
@@ -80,22 +86,28 @@ synonyms:fr:fabrique, fabriqué
 synonyms:ru:изготовление, производство
 synonyms:es:fabricación, producción, elaborado
 
+synonyms:en:fats, fat
 synonyms:fr:matières grasses, MG, gras
 synonyms:ru:жир
 
+synonyms:en:natural, of natural origin
 synonyms:fr:naturel, d'origine naturelle, origine naturelle
 synonyms:es:natural, de origen natural, origen natural
 synonyms:fi:luontainen, luonnollinen
 synonyms:it:naturale, di origine naturale, origine naturale
 
 synonyms:fr:élevé, élevée, élevés, élevées
 
+synonyms:en:from the end, after
 synonyms:fr:dès la fin, après
 
+synonyms:en:antibiotic, anti-biotic, anti-biotics, antibiotic treatment, use of antibiotics
```

Comment:
Add antibiotic**s** and antibiotic treatment**s** ?

Reply:
You should have a look at pages like http://us.openfoodfacts.org/labels or http://uk.openfoodfacts.org/labels to see what kind of labels are actually found on products.

Reply:
There is no English label in the USA except in UK: "Without antibiotics/Chickens raised without antibiotics/Porks raised without antibiotics"